modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -2000,6 +2000,15 @@ components:
                     maximum: 12
                     minimum: 1
                     type: integer
+                viewportColumnConfiguration:
```

Comment:
I would use an array (maybe called viewportColumns), and include the identifier (landscapeMobile, portraitMobile, tablet) as an additional attribute of the referenced type.

This would allow adding additional viewports without having to do changes in the YAML file.

We are doing something similar in PageFragmentInstanceDefinition for fragmentFields and widgetInstances